apps/server/src/server.ts

```diff
@@ -27,6 +27,7 @@ app.use('/category', lyricProvider.routers.category);
 app.use('/data', lyricProvider.routers.submittedData);
 app.use('/dictionary', lyricProvider.routers.dictionary);
 app.use('/submission', lyricProvider.routers.submission);
+app.use('', lyricProvider.routers.validator);
```

Comment:
this empty route looks weird, can you have here `/validator` and remove it from the router?